backend/dataall/modules/dataset_sharing/services/share_processors/s3_access_point_process_share.py

```diff
@@ -21,6 +21,7 @@ def __init__(
         target_environment: Environment,
         source_env_group: EnvironmentGroup,
         env_group: EnvironmentGroup,
+        existing_shared_buckets: bool = False
```

Comment:
is this needed in the `__init__()` - could we not handle similar to how we handle in `ProcessS3BucketShare()` when passing in `existing_shared_folders`?

Reply:
This is probably not needed at all. Will remove if not needed